claat/fetch_test.go

```diff
@@ -16,12 +16,17 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"testing"
 
+	"path/filepath"
+
+	"testing/quick"
```

Comment:
You're right, seems like my go-import doesn't always act as nicely as desired. Fixing it!